fenix/app/src/androidTest/java/org/mozilla/fenix/ui/NavigationToolbarTest.kt

```diff
@@ -142,7 +147,7 @@ class NavigationToolbarTest {
     // Test running on beta/release builds in CI:
     // caution when making changes to it, so they don't block the builds
     @Test
-    fun visitURLTest() {
+    fun basicToolbarInteractionTest() {
```

Comment:
I would delete this test entirely, we have this covered in a lot of other tests.

Reply:
As discussed, I've removed both the manual and automated test.